packages/core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -99,29 +104,68 @@ class InMemoryAnchorService extends AnchorService {
     for (const compositeKey of Object.keys(validCandidates)) {
       const candidates: Candidate[] = validCandidates[compositeKey]
 
-      let nonce = 0
-      let selected: Candidate = null
-
-      for (const candidate of candidates) {
-        const record = (await this._ceramic.ipfs.dag.get(candidate.cid)).value
-
-        let currentNonce
-        if (DoctypeUtils.isSignedRecord(record)) {
-          const payload = (await this._ceramic.ipfs.dag.get(record.link)).value
-          currentNonce = payload.header?.nonce || 0
-        } else {
-          currentNonce = record.header?.nonce || 0
+      // naive implementation for finding the "valid" commit for the document.
+      // the decision making is going to change once the anchoring service becomes aware of Doctypes
+      let selected: Candidate
```

Comment:
So, I still don't think this is exactly right for how the InMemoryAnchorService should be handling conflict resolution, but rather than continuing to hold up this PR, I went ahead and filed https://github.com/ceramicnetwork/js-ceramic/issues/664 so we can get this out the door and we can circle back to the inmemory anchor service after